### ISSUE TITLE
Author page isn't always reporting the right edited articles

### ISSUE THREAD

**Issue by jonfriskics** [orientation/orientation#90]
I edited this article: http://orientation.codeschool.com/articles/writing-on-the-blog

but my Authors profile page doesn't show it: http://orientation.codeschool.com/authors/26

**Comment by olivierlacan** [comment]
@jonfriskics Yeah currently we're only saving the last editor. It's terrible and we need to log all editors and give them all :kissing_cat: .
